codepropertygraph/src/main/scala/io/shiftleft/passes/ParallelCpgPass.scala

```diff
@@ -232,12 +241,17 @@ abstract class ConcurrentWriterCpgPass[T <: AnyRef](cpg: Cpg, outName: String =
     }
   }
 
-  private class Writer(serializedCpg: SerializedCpg, prefix: String, inverse: Boolean) extends Runnable {
+  private class Writer(serializedCpg: SerializedCpg,
+                       prefix: String,
+                       inverse: Boolean,
+                       mdc: java.util.Map[String, String] = MDC.getCopyOfContextMap)
```

Comment:
Nitpick:

For sake of consistence I would make `mdc` look the same for both `Writer`s. My personal preference would be to remove the default value here and provide it explicitly as we do in another `Writer`